### DISCUSSION TITLE
How to delete by an id and owner id?

### DISCUSSION THREAD

**Discussion by viper-42** [prisma/prisma#15195] [Q&A]
Hi I am using sessions to authenticate users to my api for a Todo list. I want users to be able to delete only their own tasks so I tried a delete with both the ownerId and id of a task but it gives an error that I can only use 1 unique field. Is this a use case for composite keys?

**Comment by pantharshit00** [comment]
Hmm, can you please share your model here? From what I can infer `ownerId` should not be unique in your case because a user should be able to create multiple todos. It will be more clear for me to understand this if you can share the model here.

**Reply by vladi-stevanovic** [reply]
👋 Hello @mitchellb613 have you perhaps had a chance to check Harshit's comment? Let us know we can help, or if you've already solved this question, we'd love to know how! It would be super useful for other users facing this problem!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
